src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/ProjectConfigurationFileChangeEventArgsTest.cs

```diff
@@ -38,7 +39,7 @@ public void TryDeserialize_RemovedKind_ReturnsFalse()
             projectInfoDeserializer: deserializerMock.Object);
 
         // Act
-        var result = args.TryDeserialize(out var handle);
+        var result = args.TryDeserialize(new TestLanguageServerFeatureOptions(), out var handle);
```

Comment:
I think there's a `TestLanguageServiceFeatureOptions.Instance` property that could be used instead of creating new instances.